app/assets/stylesheets/new_styles/_settings.scss

```diff
@@ -1,31 +0,0 @@
-/* Specific styles for the settings pages (profile, user account, privacy, services) */
-#inner_account_delete {
-	width: 700px;
-}
```

Comment:
This element doesn't exist anymore. Before it was inside of the close account facebox.